src/sourcify-chains.ts

```diff
@@ -957,10 +957,15 @@ const sourcifyChainsExtensions: SourcifyChainsExtensionsObject = {
     // Edgeware EdgeEVM Mainnet
     supported: true,
     monitored: false,
-    contractFetchAddress:
```

Comment:
Is this removal on purpose?

Reply:
nope, my fault. as well as some formatting. I will disable my VS code formatter for the workspace and push the changes. Thanks!